client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -493,36 +537,15 @@ export class PlanFeatures2023Grid extends Component<
 	}
 
 	renderPlanPrice( planPropertiesObj: PlanProperties[], options?: PlanRowOptions ) {
-		const { isReskinned, is2023OnboardingPricingGrid } = this.props;
+		const { isReskinned } = this.props;
 
-		const isLargeCurrency = planPropertiesObj.some(
-			( properties ) => properties?.rawPrice && properties?.rawPrice > 99000
+		return (
+			<PlanPricesRow
+				planPropertiesObj={ planPropertiesObj }
+				isReskinned={ isReskinned }
+				options={ options }
+			/>
```

Comment:
Extracted this into a functional component to use the `useIsLargeCurrency` hook.